editor/src/components/navigator/navigator-item/item-label.tsx

```diff
@@ -12,7 +12,7 @@ interface ItemLabelProps {
   dispatch: EditorDispatch
   target: ElementPath
   isDynamic: boolean
-  canRename: boolean
+  selected: boolean
```

Comment:
This prop doesn't appear to be used anywhere?

Reply:
Hah, I might as well just delete it then.